scrapy/loader/common.py

```diff
@@ -1,14 +0,0 @@
-"""Common functions used in Item Loaders code"""
-
-from functools import partial
-from scrapy.utils.python import get_func_args
-
-
-def wrap_loader_context(function, context):
-    """Wrap functions that receive loader_context to contain the context
-    "pre-loaded" and expose a interface that receives only one argument
-    """
-    if 'loader_context' in get_func_args(function):
-        return partial(function, loader_context=context)
-    else:
-        return function
```

Comment:
To maximize backward compatibility, we might want to deprecate this module instead of removing it. Maybe we can import the function from the new library, though.

Reply:
I guess keeping the import is better than adding a deprecation, thoughts?
Also, I removed it as it doesn't appear in the docs https://docs.scrapy.org/en/latest/topics/loaders.html, so it shouldn't be used outside `ItemLoader`, or am I missing something here?

Reply:
Yes, an import + deprecation warning. 

We're usually trying to be quite careful about deprecations, and even parts which were not publicly documented are going through deprecation process. 

Here you can also make an argument that the module is not private (not _common), and the function  is not private (not _wrap_loader_context).